pom.xml

```diff
@@ -466,6 +476,9 @@
                                 <goal>check</goal>
                             </goals>
                             <configuration>
+                                <excludes>
+                                    <exclude>**/TracerExtension*</exclude>
```

Comment:
Is it related to https://github.com/jacoco/jacoco/issues/654?

Reply:
that was my suspicion - thanks for pointing out the issue.